cpp/arcticdb/util/clock.hpp

```diff
@@ -46,6 +46,17 @@ struct LinearClock {
     }
 };
 
+struct ManualClock {
+    inline static std::atomic<entity::timestamp> time_{0};
+
+    static entity::timestamp nanos_since_epoch() {
+        return LinearClock::time_.load();
```

Comment:
Very surprised that this uses the `time_` from the other clock `LinearClock`? What is `ManualClock::time_` used for?

Reply:
This is a copy-and-paste fail. I did not expect LinearClock would quality the field name...